docs/src/stories/components/ToggleSwitch/ToggleSwitch.stories.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+
+export default {
+  title: 'Components/ToggleSwitch',
+  parameters: {
+    layout: 'padded'
+  },
+  excludeStories: ['ToggleSwitchTemplate'],
+  controls: { expanded: true },
+  argTypes: {
+    checked: {
+      control: {type: 'boolean'},
+      description: 'checkbox state'
+    },
+    disabled: {
+      description: 'disabled field',
+      control: {type: 'boolean'}
+    },
+    size: {
+      options: ['medium', 'small'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'size'
+    },
+    labelPosition: {
+      options: ['start', 'end'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'label position'
+    }
+  }
+}
+
+function classNamesForSwitch(disabled, checked, size, labelPosition) {
+  const classNames = ['ToggleSwitch'];
+
+  if (checked) {
+    classNames.push("ToggleSwitch--checked")
+  }
+  if (disabled) {
+    classNames.push("ToggleSwitch--disabled")
+  }
+  if (size === 'small') {
+    classNames.push("ToggleSwitch--small")
+  }
+  if (labelPosition === 'end') {
+    classNames.push('flex-row-reverse')
+  } else {
+    classNames.push('flex-row')
+  }
```

Comment:
I think we wanna move away from using [PCSS utilities in PVC](https://github.com/github/primer/issues/585).

The `flex-row-reverse` could be replaced with another state class like `ToggleSwitch--leftAligned`. As in, if the toggle is left aligned, then the status text will be on the right. Or be more explicit about the status's position: `ToggleSwitch--statusRight`, `ToggleSwitch--statusAtEnd`...

`flex-row` is the default, so it doesn't need to be set.

Reply:
Interesting! Will make the change 😄